docs/the-contribution-workflow.md

```diff
@@ -0,0 +1,245 @@
+## Contributing to Exercism
+
+Open source contributions can be intimidating! As you're looking around, wondering where to dive in, you might think everyone _else_ knows what's going on in the code base, or has better grasp of the language, or seems to be well known around the project.
+
+As a total beginner, where do you begin?
+
+Right here. This guide will walk you through your first open source contribution.
+
+It's going to target the smallest possible fix (improving documentation), as a "bridge" to more complex contributions later.
+
+First, go take a look at [Contributing to Exercism.io](https://github.com/exercism/exercism.io/blob/master/CONTRIBUTING.md#submitting-code-changes)
```

Comment:
We've been slowly moving all of our project-wide documentation into the http://github.com/exercism/docs repository. It is still a bit of a mess, but there's a lot of good content, and some of it is even organized fairly well.

I'd love to have this basic guide be about contributing to Exercism as a whole, and maybe use one of the language track repositories as the example instead of the main website repository (long story, but we're doing a ginormous rewrite/redesign, and the website repo is going to go away sometime in the next couple of months).

If we go down the route of having the guide be about the language tracks, then I think maybe we could link to https://github.com/exercism/docs/tree/master/contributing-to-language-tracks instead of the site contributing guide. Or there might be another document that is better, but I think the _contributing to tracks_ one is a pretty good starting point.